src/components/Filter.jsx

```diff
@@ -0,0 +1,223 @@
+          </ul>
+        </div>
+        <div className="prod-rating">
+          <div className="filter-heading">Rating</div>
+          <ul className="filter-list">
+            <li className="filter-list-item">
+              <input
+                type="radio"
+                value="4"
+                name="rating"
+                id="fourStar"
+                onClick={(e) =>
+                  filterDispatch({ type: "RATINGS", payload: e.target.value })
+                }
+                checked={state.rating === "4"}
+              />
+              <label htmlFor="fourStar">
+                <BsStarFill />
+                <BsStarFill />
+                <BsStarFill />
+                <BsStarFill />
+                <BsStarHalf />
+              </label>
+            </li>
+            <li className="filter-list-item">
+              <input
+                type="radio"
+                value="3"
+                name="rating"
+                id="threeStar"
+                onClick={(e) =>
+                  filterDispatch({ type: "RATINGS", payload: e.target.value })
+                }
+                checked={state.rating === "3"}
+              />
+              <label htmlFor="threeStar">
+                <BsStarFill />
+                <BsStarFill />
+                <BsStarFill />
+                <BsStarHalf />
+                <BsStar />
+              </label>
+            </li>
+            <li className="filter-list-item">
+              <input
+                type="radio"
+                value="2"
+                name="rating"
+                id="twoStar"
+                onClick={(e) =>
+                  filterDispatch({ type: "RATINGS", payload: e.target.value })
+                }
+                checked={state.rating === "2"}
+              />
+              <label htmlFor="twoStar">
+                <BsStarFill />
+                <BsStarFill />
+                <BsStarHalf />
+                <BsStar />
+                <BsStar />
+              </label>
+            </li>
+            <li className="filter-list-item">
+              <input
+                type="radio"
+                value="1"
+                name="rating"
+                id="oneStar"
+                onClick={(e) =>
+                  filterDispatch({ type: "RATINGS", payload: e.target.value })
+                }
+                checked={state.rating === "1"}
+              />
+              <label htmlFor="oneStar">
+                <BsStarFill />
+                <BsStarHalf />
+                <BsStar />
+                <BsStar />
+                <BsStar />
+              </label>
+            </li>
+          </ul>
+        </div>
+        <div className="prod-sorting">
+          <div className="filter-heading">Category</div>
+          <ul className="filter-list">
+            <li className="filter-list-item">
+              <input
+                name="sort"
+                checked={state.sort === "PRICE_A"}
```

Comment:
Use more descriptive state values, PRICE_A can be improved

Reply:
Ok I will